subsys/bluetooth/audio/bap_broadcast_sink.c

```diff
@@ -798,31 +660,33 @@ static bool pa_decode_base(struct bt_data *data, void *user_data)
 		}
 	}
 
+	base_size = bt_bap_base_get_size(base);
+
 	/* Store newest BASE info until we are BIG synced */
 	if (sink->big == NULL) {
+		ret = bt_bap_base_get_pres_delay(base);
+		if (ret < 0) {
+			LOG_DBG("Could not get presentation delay: %d", ret);
+			return ret;
+		}
```

Comment:
I think this should be an assert. 

We have 
```
/* Base is NULL if the data does not contain a valid BASE */
	if (base == NULL) {
		return true;
}
```

So the BASE has been validated at this point, so we can assume that it contains a valid presentation delay

Reply:
Or if already validated, we can remove the check?

Reply:
`(void)bt_bap_base_get_pres_delay(base)` is also an option yeah. An assert has the advantage that if something breaks, it breaks more loudly :D